cleanup.sh

```diff
@@ -1,3 +1,3 @@
-rm -R x86_64/
+rm -R arm64/
```

Comment:
you can probably still also keep the x86_64 part here bc some ppl will run on linux